build.rs

```diff
@@ -195,24 +174,16 @@ fn cross_compiling() -> Result<Option<CrossCompileConfig>> {
 ///
 /// (hrm, this is sort of re-implementing what distutils does, except
 /// by passing command line args instead of referring to a python.h)
-static SYSCONFIG_FLAGS: [&str; 7] = [
-    "Py_USING_UNICODE",
-    "Py_UNICODE_WIDE",
+type BuildFlag = &'static str;
```

Comment:
Is this *really* so meaningful?
I think maybe `struct BuildFlags(HashSet<&'static str>)` would be a better abstraction.
Then we can move `BUILD_FLAGS` into `impl BuildFlags` block.